workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/views/movies.py

```diff
@@ -18,7 +18,7 @@
 import vcm.catalog
 
 from fv3net.diagnostics.prognostic_run import config
-import fv3net.diagnostics.prognostic_run.load_diagnostic_data as load_diags
+import fv3net.diagnostics.prognostic_run.load_run_data as load_diags
```

Comment:
refactored `load_diagnostic_data` to `load_run_data` since the `diagnostic` word is overloaded and here mostly means reduced diagnostics that are computed from the raw "run" output